js/packages/api/index.d.ts

```diff
@@ -5,7 +5,6 @@ import pb from './root.pb'
 import account = pb.berty.account.v1
 import messenger = pb.berty.messenger.v1
 import protocol = pb.berty.protocol.v1
-import types = pb.berty.types.v1
 import errcode = pb.berty.errcode
 
-export { account, messenger, protocol, types, errcode }
```

Comment:
can you remove this pls ? what made this refactor easy was that I only allowed the root import "beapi" in my previous refactor